Scripts/pwTypeToggle.js

```diff
@@ -0,0 +1,21 @@
+const pwTextTypeToggleBtnList = document.querySelectorAll('.pwTextTypeToggleBtn');
+
+
+// 비밀번호 숨김 구현
+function pwVisualTypeToggleBtn(e) {
+   if (!e.target.classList.contains('pwTextInputIsNotHidden')) {
+      e.target.classList.add('pwTextInputIsNotHidden');
+      e.target.previousElementSibling.setAttribute('type','text');   
+      return
+   } 
+   e.target.classList.remove('pwTextInputIsNotHidden');
+   e.target.previousElementSibling.setAttribute('type','password');
+};
```

Comment:
변수를 선언해 코드의 가독성을 높이면 좋겠습니다.

```suggestion
function pwVisualTypeToggleBtn(e) {
   const isShowPassword = !e.target.classList.contains('pwTextInputIsNotHidden')
   const toggleBtn =  e.target.previousElementSibling;
   
   if (isShowPassword) {
      e.target.classList.add('pwTextInputIsNotHidden');
      toggleBtn.setAttribute('type','text');   
      return
   } 
   
   e.target.classList.remove('pwTextInputIsNotHidden');
   toggleBtn.setAttribute('type','password');
};
```

Reply:
오... 참고하겠습니다... 감사합니다!